src/cpu/aarch64/jit_uni_batch_normalization.cpp

```diff
@@ -0,0 +1,2428 @@
+/*******************************************************************************
+* Copyright 2017-2020 Intel Corporation
```

Comment:
[here and other places]

Since it's newly created file so I think just "Copyright 2020" is enough.

Reply:
Thank you. I fixed them and force pushed.